daemon/cluster/executor/container/controller.go

```diff
@@ -19,6 +14,10 @@ import (
 	"github.com/pkg/errors"
 	"golang.org/x/net/context"
 	"golang.org/x/time/rate"
+	"os"
```

Comment:
Can you put these back at the top? Convention in this repo is to group imports in "standard library" imports and external imports